lib/view_helpers.py

```diff
@@ -34,8 +39,10 @@ def is_query_after_new_line(view, current_location):
 
 def should_return_empty_list(view, locations, prefix):
     last_command_insert_snippet = view.command_history(-1)[0] == "insert_snippet"
+    wrong_view = sublime.active_window().active_view().id() != view.id()
```

Comment:
you can use your own active_view function here...

Reply:
👍